src/rastervision/ml_tasks/semantic_segmentation.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+from rastervision.core.ml_task import MLTask
+from rastervision.core.box import Box
+
+
+class SemanticSegmentation(MLTask):
+
+    def get_train_windows(self, scene, options):
+        raster_source = scene.raster_source
+        meta = raster_source.image_dataset.meta
```

Comment:
Could use RasterSource.get_extent() instead

Reply:
Thank you